app/services/report/group_by.rb

```diff
@@ -0,0 +1,52 @@
+# frozen_string_literal: true
+
+module Report
+  class GroupBy < ApplicationService
+    attr_reader :group_by_field
+
+    POSSIBLE_GROUP_BY_INPUTS = ["team_member", "client", "project", "week"]
```

Comment:
```suggestion
    POSSIBLE_GROUP_BY_INPUTS = ["team_member", "client", "project", "week"].freeze
```
cc: @keshavbiswa @rohitjoshixyz